pytorch/DCCComputation.py

```diff
@@ -54,7 +58,12 @@ def computeHyperParams(pairs, Z):
     numpairs = len(pairs)
     numsamples = len(Z)
     epsilon = np.linalg.norm(Z[pairs[:, 0].astype(int)] - Z[pairs[:, 1].astype(int)], axis=1)
-    epsilon = np.sort(epsilon[np.where(epsilon / np.sqrt(cfg.DIM) > cfg.RCC.NOISE_THRESHOLD)])
+    epsilon = np.sort(epsilon / np.sqrt(cfg.DIM))
+    # largest is noise just consider as noise
+    if epsilon[-1] < cfg.RCC.NOISE_THRESHOLD:
+        epsilon = np.asarray([cfg.RCC.NOISE_THRESHOLD])
+    else:
+        epsilon = epsilon[np.where(epsilon > cfg.RCC.NOISE_THRESHOLD)]
```

Comment:
In the original code the 'epsilon' used was never factored by 1/np.sqrt(cfg.DIM). This factor was only used for selection. Can you please make this changes ?

Reply:
Ok changed back